docs/kata-containers.md

```diff
@@ -0,0 +1,49 @@
+# Kata containers support in CRI-RM
+
+Kata containers represent an alternative runtime to Runc containers,
+and are conceptually defined by a containers which replace their sandbox 
+with a lightweight Virtual Machine.
+
+In order to enable Kata container in a Kubernetes-CRI-RM stack, both Kubernetes 
+and the Container Runtime Engine need to be aware of the new runtime environment:
+
+  * The Container Runtime Engine can only be CRI-o or containerd, and will need to
+   have the runtimes enabled in their configuration files.
+  * Kubernetes will also have to be made aware via a "RuntimeClass"  resource
+
+After these prerequesites are satisfied, the configuration file for the target  Kata Container, will have to have the flag "SandboxCgroupOnly" to true. As of Kata 2.0 this is the only way Kata containers can with the Kubernetes cgroup naming conventions.
```

Comment:
typos: prerequisites... "SandboxCgroupOnly" set to true... can work with the Kubernetes cgroup naming conventions.